marketplace_builder/views/pages/release-notes/2018-07-30.liquid

```diff
@@ -0,0 +1,37 @@
+---
+converter: markdown
+metadata:
+  title: 2018 07 30 — Release Notes
+  description: 2018-07-30 - New liquid tags, deprecate render_form tag, deprecate log graphql mutation
+slug: release-notes/2018-07-30
+searchable: true
+---
+### {{ page.metadata.description }}
+
+## Improvements
+
+* **include_form tag**: new tag allows you to use normal liquid syntax to pass variables
+
+  {% raw %}
+  ```
+  {% render_form 'edit_user', resource_id: @user.id %}
```

Comment:
can we deprecate render_form ?

Reply:
yes - deprecation doesnt mean removal. yet. ;)